### PR TITLE
Update model.js

### PR DESCRIPTION
**Pull request by dhatcher**
buffer by the difference of min/max to keep from having issues with x or y being zero.

### CODE REVIEW

File: src/model.js [hunk 1]
```diff
@@ -396,10 +396,12 @@ module.exports = function(koop) {
...
-        ymin = ymin-Math.abs(ymin*buffer);
-        xmax = xmax+Math.abs(xmax*buffer);
-        ymax = ymax+Math.abs(ymax*buffer);
+        let ydiff = ymax-ymin;
+        let xdiff = xmax-xmin;
+        xmin = xmin-Math.abs(xdiff*buffer);
+        ymin = ymin-Math.abs(ydiff*buffer);
+        xmax = xmax+Math.abs(xdiff*buffer);
+        ymax = ymax+Math.abs(ydiff*buffer);
         return {xmin:xmin, ymin:ymin, xmax:xmax, ymax:ymax};
```
Comment (rgwozdz) [inline src/model.js:405]: for simplicity, you can just return:

`return { xmin, ymin, xmax, ymax };`

**Review by rgwozdz — APPROVED** [review]

**Comment by rgwozdz** [comment]
Can you add a `Unreleased` entry to changelog, with a subentry `Changed`?

File: src/model.js (hunk 1 above)
Comment (dhatcher) [reply src/model.js]: done!

**Comment by dhatcher** [comment]
Updated change log

**Pull request merged**